.claude/skills/todoapp-pipeline/phases/phase0-trigger-detection.md

```
@@ -0,0 +1,56 @@
+# Bug Report: ← qa 扱い
+
+# QA: ... ← qa 扱い
+```
+
+判別不能の場合は人間に確認する。
+
+## 対話モード
+
+引数なしで起動された場合：
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)